### DISCUSSION TITLE
Frage zu Punkten im Changelog zur Version 1.26.0 (2023-01-02)

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#819] [Q&A] [answered]
Da ich jetzt immer fleißig die Changelogs lese habe ich Fragen zu einigen Punkten. 

- Add un_ignore list to central config
- Allow to disable un_ignore load

Hat das was mit `/config/homematicip_local/unignore` zu tun? 
Hab jetzt mal geschaut ob sich irgendwo was einstellen lässt, aber nix gefunden.

- Allow to disable cache

Was hat es damit auf sich und wo kann man das einstellen?
(Da ich auf meinem Testsystem häufig was ändere käme mir kein Caching ganz recht)

Danke schon mal für Infos.

**Comment by SukramJ** [comment, marked as answer]
Das sind Änderungen an der API von HAHomematic(BackendLib):

> Add un_ignore list to central config

Ich hoffe das die Konfiguration für unignore mal im Config-Flow gemacht werden kann, aber aktuell fehlt mir da noch eine sinnvolle Umsetzungsidee. Mittelfristig möchte ich von der Text-Datei weg.

> Allow to disable un_ignore load
Allow to disable cache
Add un_ignore list to central config

Ich arbeite gerade an automatisierten Test für HAHomematic. Da sind entsprechende Optionen sinnvoll. Eine Umsetzung in der UI ist nicht geplant.
Eigentlich sollte das Caching (paramset_descriptions, device_descriptions) auf Testsystemen aber kein Problem sein.

**Reply by Baxxy13** [reply]
Danke für die Info's.

Unignore direkt über Config-Flow zu konfigurieren klingt gut. Ne passable Idee dazu habe ich leider aktuell nicht.
Kenne aber auch die Möglichkeiten nicht die man eventuell nutzen könnte.
